src/full/Agda/TypeChecking/Monad/Base.hs

```diff
@@ -4660,6 +4671,13 @@ data TypeError
         | TriedToCopyConstrainedPrim QName
           deriving (Show, Generic)
 
+type DataOrRecordE = DataOrRecord' InductionAndEta
+
+data InductionAndEta = InductionAndEta { recordInduction   :: Maybe Induction
+                                       , recordEtaEquality :: EtaEquality
+                                       }
+  deriving (Show, Generic)
```

Comment:
Style suggestion for such records: line break after the constructor and then only 2 spaces indentation for the list of fields.

See the (unfortunately somewhat outdated) style-guide at https://github.com/andreasabel/haskell-style-guide/blob/master/haskell-style.md#data-declarations